lib/pbench/agent/tool_meister.py

```diff
@@ -1725,7 +1725,8 @@ def sysinfo(self, data: Dict[str, str]) -> int:
         try:
             with o_file.open("w") as ofp, e_file.open("w") as efp:
                 my_env = os.environ.copy()
-                my_env["pbench_install_dir"] = self.pbench_install_dir
+                my_env["sysinfo_install_dir"] = self.pbench_install_dir
+                my_env["sysinfo_full_hostname"] = self._params.hostname
```

Comment:
Not really complaining; however, since we control both sides of the interface, and you don't want to use the "standard" environment variables here, why not simply add these values to the `pbench-sysinfo-dump` argv instead of creating/reading new environment variables?

Reply:
I thought about that, but wanted to reduce the changes as much as possible to the non-`pbench-sysinfo-dump` code.